### PR TITLE
SEO changes for example scenarios

### PR DESCRIPTION
**Pull request by alexbuckgit**

### CODE REVIEW

File: docs/example-scenario/apps/devops-dotnet-webapp.md [hunk 1]
```diff
@@ -79,7 +80,7 @@ This [decision tree for Azure compute services](/azure/architecture/guide/techno
 
 ### Prerequisites
 
-- You must have an existing Azure account. If you don't have an Azure subscription, create a [free account][azure-free-account] before you begin.
+- You must have an existing Azure account. If you don't have an Azure subscription, create a [free account](https://azure.microsoft.com/free) before you begin.
```
Comment (MikeWasson) [inline docs/example-scenario/apps/devops-dotnet-webapp.md]: Are we supposed to use the query string to track clicks?

File: docs/example-scenario/data/hybrid-etl-with-adf.md [hunk 2]
```diff
@@ -1,5 +1,6 @@
 ---
-title: Hybrid ETL with existing on-premises SSIS and Azure Data Factory
+title: Hybrid ETL with existing on-premises SSIS and ADF
```
Comment (MikeWasson) [inline docs/example-scenario/data/hybrid-etl-with-adf.md]: Per style guide ADF should always be spelled out

File: docs/example-scenario/apps/devops-dotnet-webapp.md (hunk 1 above)
Comment (alexbuckgit) [reply docs/example-scenario/apps/devops-dotnet-webapp.md]: Fixed.

File: docs/example-scenario/data/hybrid-etl-with-adf.md (hunk 2 above)
Comment (alexbuckgit) [reply docs/example-scenario/data/hybrid-etl-with-adf.md]: Fixed.

**Pull request merged**